Exercise_2.java

```diff
@@ -1,3 +1,5 @@
+//Time Complexity = O(1)
+//Space Complexity = O(1)

```

Comment:
Please check the mentioned space complexity

Reply:
Please try to mention the time and space complexity for each function instead of problem as a whole since this is an implementation problem. If you consider for single functions, then the mentioned TC and SC is correct.